mp/src/game/shared/gamemovement.cpp

```diff
@@ -3931,26 +3931,34 @@ void CGameMovement::CheckFalling( void )
 	player->m_Local.m_flFallVelocity = 0.0f;
 }
 
-void CGameMovement::PlayerRoughLandingEffects( float fvol )
+void CGameMovement::PlayerRoughLandingEffects(float fvol, short surfacePropsIndex)
```

Comment:
When there's a default value it's good to put it in an inline comment so people don't have to constantly switch between header and cpp file:

```suggestion
void CGameMovement::PlayerRoughLandingEffects(float fvol, short surfacePropsIndexOverride /*= -1*/)
```

Reply:
I've never seen this, I like it.